Example/Example Swift/ResultTableViewController.swift

```diff
@@ -16,12 +16,12 @@ import Gini_iOS_SDK
 final class ResultTableViewController: UITableViewController {
     
     /**
-     The result dictionary from the analysis process.
+     The result collection from the analysis process.
      */
-    var result: [String: GINIExtraction]!
-    
-    fileprivate var sortedKeys: [String] {
-        return Array(result.keys).sorted(by: <)
+    var result: [Extraction]! {
```

Comment:
Hmmm... why the implicitly unwrapped optional?

Reply:
🤷‍♂️ No idea to be honest, maybe I _copied-pasted_ and didn't notice it